src/main/scala/com.snowplowanalytics.maxmind.iplookups/IpLookups.scala

```diff
@@ -14,11 +14,10 @@ package com.snowplowanalytics.maxmind.iplookups
 
 import java.io.File
 import java.net.InetAddress
-
```

Comment:
yes sorry if my explanation wasn't complete earlier but the imports are grouped in this manner: jdk, other libraries, our libraries

Reply:
oh right I saw now from other project's imports how imports are structured :+1: